apps/web/features/feedback-file-upload/file-upload-dropzone.tsx

```diff
@@ -30,7 +30,7 @@ function FileUploadDropzone() {
     isLoading,
     files,
     handleDeleteSingleFile,
-  } = useFileUpload();
+  } = useFileUpload(workspaceId as string);
```

Comment:
Useless type assigning

```suggestion
  } = useFileUpload(workspaceId);
```